### ISSUE TITLE
(MAL) server offline

### ISSUE THREAD

**Issue by TheGreatT20** [MALSync/MALSync#677]
it is showing that the Mal server is offline whenever I try to update my mal from mal sync extension. It occurs whenever I try to update anything on mal. Please address this

**Comment by henrik9999** [comment]
MAL-Sync currently has problems with MyAnimeList. Please switch to 'MyAnimeList API (BETA)' in the settings for now. List mirroring does not work from and to MAL. We are looking into solving the problem if even possible.

**Comment by MKRhere** [comment]
Switching to MyAnimeList API and trying to authenticate says "Extension or userscript not installed." (Firefox Developer Edition 88.0b9 on Debian Linux)

**Comment by lolamtisch** [comment]
Yesterday I debugged a similar problem. But we did not find the culprit. For him it did work as long first party insulation was on, what does not make any sense as it should not rely on cookies. It only saves a value in the session storage during the authentication for 10 seconds it needs it.
After a lot of tinkering with privacy settings apparently only reinstalling the extension fixed it. Something strange Is happening on Firefox
